### PR TITLE
update btc tracker to use coinbase

### PR DESCRIPTION
**Pull request by tych0**
Seeems btc-e has died, so that was a bad choice. Coinbase is much more
reputable, perhaps they'll stick around for a few more years.

Note that this drops quite a bit of the flexibility of the previous widget
w.r.t. other currencies, but adds support for many more fiat currencies.
This is simply because of the exchange capabilities. However, I've taken
care to make sure that old configs will at least not crash the new widget,
with a best-effort to honor the ones that can be honored.

Signed-off-by: Tycho Andersen <tycho@tycho.ws>

### CODE REVIEW

File: libqtile/widget/bitcoin_ticker.py [hunk 1]
```diff
@@ -30,52 +30,39 @@
...
-        widget.BitcoinTicker(format="BTC: {avg}")
-
-        # display the average price of litecoin in local currency
-        widget.BitcoinTicker(format="LTC: {avg}", source_currency='ltc')
+        widget.BitcoinTicker()
 
-        # display the average price of litecoin in bitcoin
-        widget.BitcoinTicker(format="BTC: ฿{avg}", source_currency='ltc', currency='btc', round=False)
+        # display it in Euros:
+        widget.BitcoinTicker(currency=EUR)
```
Comment (flacjacket) [inline libqtile/widget/bitcoin_ticker.py]: Should this be a string `"EUR"`?

File: libqtile/widget/bitcoin_ticker.py [hunk 2]
```diff
@@ -30,52 +30,39 @@
...
-        if 'error' in body and body['error'] == "invalid pair":
-            locale.setlocale(locale.LC_MONETARY, "en_US.UTF-8")
-            self.currency = locale.localeconv()['int_curr_symbol'].strip()
-            body = self.fetch(self.url)
-        for k, v in body['ticker'].items():
-            formatted[k] = v
-            if self.round:
-                formatted[k] = locale.currency(v)
-        return self.format.format(**formatted)
+        return "BTC: {symbol}{amount}".format(symbol=self.symbol, amount=body['data']['amount'])
```
Comment (flacjacket) [inline libqtile/widget/bitcoin_ticker.py]: This should be left as a configurable

Comment (flacjacket) [inline libqtile/widget/bitcoin_ticker.py]: Oh, wait, nevermind, I see, it's just for bitcoin to a fiat currency.

File: libqtile/widget/bitcoin_ticker.py (hunk 1 above)
Comment (tych0) [inline libqtile/widget/bitcoin_ticker.py]: Yep, sorry, fixed :)

**Comment by flacjacket** [comment]
Great, thanks for the update!

**Pull request closed without merging**